.github/workflows/checks.yml

```diff
@@ -0,0 +1,100 @@
+name: Check
+
+on:
+  # Curently, this workflow is triggered by pull request events only.
+  # Uncomment the following line to run this workflow on push events
+  # push:
+  #   branches:
+  #     - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  flutter-analyze:
+    name: flutter analyze
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: subosito/flutter-action@v2
+        with:
+          flutter-version: '3.10.6' # Uses the minimum supported version.
```

Comment:
would it be possible to parse this value from `pubspec.yaml`?

Reply:
Not sure but maybe a script to read the `pubpec.yaml` file and parse this value or we could use GitHub Action environmental variable  but not sure it would be worth it This only  updated quarterly or so.

Reply:
Added a yaml variable instead from [this](https://docs.geoserver.org/2.21.x/en/user/styling/ysld/reference/variables.html)

Reply:
Thank <3 LGTM!